### DISCUSSION TITLE
Where is the Swell Community plan?

### DISCUSSION THREAD

**Discussion by efoken** [swellstores/community#284] [Help] [answered]
I had a 2 weeks trial and now I am missing the Community plan, where you pay 2% per transaction instead of a monthly fee. Where is it? Is it not anymore available?

I just want to re-launch a small shop and I am willing to pay 2% per transaction, but not 19$ monthly, because I just have about 200$ online sales per month. So I will probably stick with WordPress and use WPGraphQL ...

**Comment by ericingram** [comment, marked as answer]
We did deprecate the Community plan recently and will send an official announcement with more details soon. All plans now include general support and a dedicated test environment.

Totally understandable if the current plans are not suitable for your use case. We hope to change that with the upcoming Apps marketplace. Please check back in the future as the shop grows!

**Reply by ralphilius** [reply]
What will happen to current users?

**Reply by ericingram** [reply]
Current users can keep the plan.
